content/github/getting-started-with-github/quickstart/fork-a-repo.md

```diff
@@ -18,7 +18,7 @@ topics:
 ---
 ### About forks
 
-Most commonly, forks are used to either propose changes to someone else's project or to use someone else's project as a starting point for your own idea.
+Most commonly, forks are used to either propose changes to someone else's project or to use someone else's project as a starting point for your own idea. You can fork a repository to create a copy of the repository and make changes without affecting the upstream repository. For more information, see "[Working with forks](/github/collaborating-with-issues-and-pull-requests/working-with-forks)."
```

Comment:
I made a few changes to the intro here and moved the link to "[Working with forks](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/working-with-forks)" to the intro here.

In the GitHub docs, we don't use headers for links so I was looking for another place to put the link and it seems to make sense here if people have more questions about what to do with forks when they're reading the beginning of this article.

Let me know if you have any questions about these changes ✏️

Reply:
That certainly make sense.